### ISSUE TITLE
Cyrillic and Greek too?

### ISSUE THREAD

**Issue by EbenSorkin** [source-foundry/glyph-filter-lists#4]
This is great! I wonder if there is an appetite  for a similar set of lists to be made for Cyrillic and Greek.

**Comment by chrissimpkins** [comment]
Absolutely!   Will take any new sets that are felt to be of use out there.  Are you interested in contributing them Eben?  If so, please let me know if you have any questions about the format or there is anything that I can do to make it happen.

**Comment by EbenSorkin** [comment]
I'll solicit some lists - it may be that pyfontaine has what we need for
these already.

It doesn't look like these lists used pyfontaine. But is that right?

**Comment by chrissimpkins** [comment]
I didn't get any of this information from pyfontaine.  The sets here currently include Adobe, Google, and Underware defined char set spec data.  If you come across something useful in pyfontaine let me know.  We may be able to programmatically pull the info into this text file format instead of doing it by hand.

**Comment by chrissimpkins** [comment]
@EbenSorkin perhaps these are what you are looking for?  https://github.com/googlefonts/gftools/tree/master/Lib/gftools/encodings/GF%20Glyph%20Sets

**Comment by EbenSorkin** [comment]
Those are Great! But no. I was thinking about Having Cyrillic coverable being more atomic and more like Latin by offering language sets like this:

https://scriptsource.org/cms/scripts/page.php?item_id=script_detail&key=Cyrl

I am not sure how happy they would be having glyphs filter lists made from these or maybe they would be VERY happy. I/we could ask. What do you think?

-e.

**Comment by chrissimpkins** [comment]
What a great web site!  It looks like we may need to clarify the licensing before we use any of those data.  The footer says that the site is under CC-by-SA 3.0 but also points to other sources (e.g., SIL Ethnologue text) for the language data.

**Comment by EbenSorkin** [comment]
Agreed. I know one of the maintainers. Shall I ask him what he thinks?

-e.

**Comment by chrissimpkins** [comment]
That would be great Eben.  Someone with a Github account who would be willing to weigh in this thread?

**Comment by jvgaultney** [comment]
Hi - I'm part of the original ScriptSource team and involved with developing the [terms of use](http://scriptsource.org/ToU). Since the list of languages that use a particular script is tightly integrated with the data from the Ethnologue, I suspect (but cannot authoritatively judge) that you would need to abide by the terms of the [Ethnologue Terms of Use](https://www.ethnologue.com/terms-use) and apply for permission to use the data in this project. 

I would love to see all the data on ScriptSource (and Ethnologue) released under a clear open data license, but that's not possible at this time. The cost to SIL of maintaining the Ethnologue and related data sets is significant, and yet we make as much of that data available without charge as we can, particularly to language communities. In other cases we look to organizations and companies (like Google) to license the data for their commercial projects.

If you want to explore licensing the data, follow the links on the Ethnologue ToU page, send them an email. You can cc me if you like, to victor_gaultney at the SIL domain.

**Comment by EbenSorkin** [comment]
My interpretation of the terms suggest that indeed whoever wanted to make
Glyphs app filter lists would need to license the data in some way and even
then maybe not allow the list to be downloaded.

Did I come to a correct understanding?

-e.

**Comment by chrissimpkins** [comment]
Thanks for the input Victor.  I really appreciate it.  IMO, we care less about redistributing SIL data from this repository and more about finding a way to make  the data accessible to those who want to use it for glyph set filtering purposes in font editors.  Would there happen to be any interest in the development of SIL developed / SIL maintained newline delimited list formatted data that can be used as font editor filters by type developers who apply for access to the data under your terms of use?  The goal of this repository is to centralize the distributed sites of data that can be used for this purpose.  For data that cannot be redistributed here, we might consider pointing to the source and provide instructions on  how to access it.  We could also develop tools that turn serialized data (e.g., JSON) into properly formatted newline delimited lists for data sources that provide an API.  If this is something of interest, let us know how we can help to get the data into type developers hands.

**Comment by chrissimpkins** [comment]
Circling back to this issue.  If there is any interest in additional writing system support and anyone is aware of freely licensed data sources that allow us to redistribute the data for font editors here, please let me know.

**Comment by EbenSorkin** [comment]
I am thinking of adding data from public sources for Pan-African Latin, and
some additional Cyrillic-based languages.

**Comment by mhosken** [comment]
You'll find much of use here: https://github.com/silnrsi/sldr (sldr/ subdir) with generous licensing (although a source credit might be nice). I recommend you think through which of the exemplars you want to use. Alternatively you might ingest LDML directly. That's the general format that is being used for this stuff, by us and CLDR. The gclusters subdir may also be of interest. FWIW the SLDR is an occasional (about annual) superset of CLDR.

**Comment by chrissimpkins** [comment]
This looks fantastic @mhosken.  Thank you very much!

> Alternatively you might ingest LDML directly.

This is an interesting idea.  Do you offer public API access to the XML files or are you suggesting access directly via the GitHub raw file URL in your repository?  We could come up with something that converts exemplar string data to glyph name data where glyph name data are available.  We are currently using Glyphs app GlyphData.xml defined glyph names in this project and the goal is to turn information about these scripts/writing systems into something that can be used by glyph filtering / counting processes within font editors.
